Configuration/Generator/python/WToLNu_14TeV_TuneCUETP8M1_cfi.py

```diff
@@ -0,0 +1,29 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I don't think we need to have old Tune. This file should be deleted.

Reply:
I'm going to do it